src/SIL.XForge.Scripture/ClientApp/.storybook/main.ts

```diff
@@ -6,7 +6,7 @@ module.exports = {
     options: {}
   },
   docs: {
-    autodocs: true
+    autodocs: 'tag'
```

Comment:
The autogenerated docs page shows each story on the same page. This works OK unless your stories all open dialogs, in which case they all open at once on top of each other, and it just makes a mess. Hence turning it off, except when specifically turned on by specifying the autodocs tag.